src/SymfonyCache/UserContextListener.php

```diff
@@ -104,9 +104,9 @@ public function preHandle(CacheEvent $event)
                 return;
             }
 
-            // We must not call $request->isMethodSafe() here, because this will mess up with later configuration of the `http_method_override` that is set onto the request by the Symfony FrameworkBundle.
+            // We must not call $request->isMethodCacheable() here, because this will mess up with later configuration of the `http_method_override` that is set onto the request by the Symfony FrameworkBundle.
             // See http://symfony.com/doc/current/reference/configuration/framework.html#configuration-framework-http-method-override
-            if (in_array($request->getRealMethod(), ['GET', 'HEAD', 'OPTIONS', 'TRACE']) && $hash = $this->getUserHash($event->getKernel(), $request)) {
+            if (in_array($request->getRealMethod(), ['GET', 'HEAD']) && $hash = $this->getUserHash($event->getKernel(), $request)) {
```

Comment:
i think this is what we meant to do. @Toflar do you agree or did i somehow misinterpret what we do? we want to know if the request method is cacheable. OPTIONS and TRACE usually are not cacheable.

Reply:
I agree, `OPTIONS` and `TRACE` must not be cached acccording to spec. Perfect 👍